platform/mellanox/mlnx-platform-api/tests/test_thermal_policy.py

```diff
@@ -159,6 +185,44 @@ def test_all_fan_presence_condition():
     fan_info.collect(chassis)
     assert condition.is_match({'fan_info': fan_info})
 
+def test_any_fan_fault_condition():
```

Comment:
should you consider to have a test file for the tests instead of the general code used for production? not must but i think it is better